script/extract-json.mjs

```diff
@@ -0,0 +1,239 @@
+
+import { spawn } from 'node:child_process';
+import { rm } from 'node:fs/promises';
+import { join } from 'node:path';
+import { writeFileSync } from 'node:fs';
+
+const repoUrl = 'https://github.com/HadrienGardeur/web-speech-recommended-voices.git';
+const repoBranch = 'locales-for-voice-names';
+const repoPath = 'script/web-speech-recommended-voices';
+const jsonFilePath = join(repoPath, 'path/to/your/json/file.json');
+
+// Clone the repository
+await new Promise((resolve, reject) => {
+  const cloneProcess = spawn('git', ['clone', '--branch', repoBranch, repoUrl, repoPath]);
```

Comment:
The cloning process should use `--depth=1` to increase efficiency of the clone, since it's only being used for a single build before being deleted. See: https://github.blog/open-source/git/get-up-to-speed-with-partial-clone-and-shallow-clone/

Reply:
fixed by https://github.com/readium/speech/pull/7/commits/0e1165aa5376a573f59b6bbdf92eb88e76716ad0